Userland/Applications/Spreadsheet/SpreadsheetModel.cpp

```diff
@@ -103,6 +103,36 @@ GUI::Variant SheetModel::data(const GUI::ModelIndex& index, GUI::ModelRole role)
         return {};
     }
 
+    if (to_underlying(role) == to_underlying(Role::Tooltip)) {
+        auto const* cell = m_sheet->at({ (size_t)index.column(), (size_t)index.row() });
+        if (!cell || !cell->thrown_value().has_value())
+            return {};
+
+        auto value = cell->thrown_value().value();
+        if (value.is_object()) {
+            auto& object = value.as_object();
+            if (is<JS::Error>(object)) {
+                auto& error = static_cast<JS::Error&>(object);
```

Comment:
could use early return style here to make it a bit nicer

```cpp
if (!value.is_object())
    return {};
auto& object = value.as_object();
if (is<JS::Error>(object))
    return {};
auto& error = static_cast<JS::Error&>(object);
...
```

Reply:
It's a copy of earlier code, but yeah, I like this better, will fix the old one too.